runelite-client/src/main/java/net/runelite/client/plugins/minimap/MinimapPlugin.java

```diff
@@ -187,4 +255,80 @@ private void restoreOriginalDots()
 
 		System.arraycopy(originalDotSprites, 0, mapDots, 0, mapDots.length);
 	}
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.MINIMAP_GROUP_ID)
+		{
+			if (config.leftAlignOrbs() != AlignmentMode.OFF)
+			{
+				if (config.leftAlignOrbs() == AlignmentMode.BOTH)
+				{
+					moveOrbs();
+				}
+				else if (!client.isResized() && config.leftAlignOrbs() == AlignmentMode.FIXED)
+				{
+					moveOrbs();
+				}
+				else if (client.isResized() && config.leftAlignOrbs() == AlignmentMode.RESIZEABLE)
+				{
+					moveOrbs();
+				}
+			}
+		}
+	}
+
+	private void moveOrbs()
+	{
+		for (int i = 0; i < orbs.size(); i++)
+		{
+			Widget widget = client.getWidget(orbs.get(i));
+			if (widget != null)
+			{
+				//Back up default widget positions before they are changed
+				if (orbDefaultPositions[i][0] == -1)
+				{
+					orbDefaultPositions[i] = new int[]{
+						widget.getOriginalX(),
+						widget.getRelativeX(),
+						widget.getOriginalY(),
+						widget.getRelativeY()};
+				}
+				widget.setOriginalX(1);
+				widget.setRelativeX(1);
+
+				widget.setOriginalY(((i + 1) * 32) + 3);
+				widget.setRelativeY(((i + 1) * 32) + 3);
```

Comment:
Are the 32 and 3 height and margin?

Reply:
32 is the height of each orb, including any margin they may have. 3 is the initial offset so the health orb doesn't start covering the XP Tracker orb.

Reply:
Store them 🤔